README.md

```diff
@@ -222,14 +225,31 @@ A man page can be generated at run-time using the
 [man](https://github.com/WGUNDERWOOD/tex-fmt/tree/main/man)
 directory for more details.
 
+### Run before every commit
+
+You can format your LaTeX files before every commit using
+[pre-commit](http://pre-commit.com) with the following `.pre-commit-config.yaml`
+in your repository root:
+
+```yaml
+repos:
+  - repo: https://github.com/WGUNDERWOOD/tex-fmt
+    rev: v0.5.3
```

Comment:
This should be the actual tag when this feature is released

Reply:
Agreed, I will check this going forward.